pkg/controller/ambassadorinstallation/reconcile.go

```diff
@@ -279,11 +306,13 @@ func (r *ReconcileAmbassadorInstallation) Reconcile(request reconcile.Request) (
 					Kind:    "AuthService",
 				}, request.Namespace)
 				if err != nil {
-					log.Error(err, "Could not look up AuthService in the cluster")
+					message := "Could not look up AuthService in the cluster"
```

Comment:
In all these error handling code paths, was it intended for the extracted `message` to also be reported using `r.Report()`?

Reply:
yes, thanks!